website/public/js/controllers/groupsCtrl.js

```diff
@@ -47,6 +47,11 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
       group._editing = false;
     };
 
+    $scope.cancelSave = function(group) {
+      group = group.$get();
```

Comment:
According to the resource docs (
https://docs.angularjs.org/api/ngResource/service/$resource) it returns an
empty object that is populated on result which will trigger rendering
changes. I guess this could create some interesting client experience, but
I wasn't sure if going with something like a shadow copy directive would
over complicate the change.

Also I am new to Angular and Jade, so there is likely an even better way to
handle this.

On Fri, Dec 25, 2015, 20:32 Blade Barringer <notifications@github.com>
wrote:

> In website/public/js/controllers/groupsCtrl.js
> <https://github.com/HabitRPG/habitrpg/pull/6400#discussion_r48444133>:
>
> > @@ -47,6 +47,11 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
> >        group._editing = false;
> >      };
> >
> > +    $scope.cancelSave = function(group) {
> > +      group = group.$get();
>
> Does the $get method return the group object? I thought group.$get
> returned a promise.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/6400/files#r48444133>.
>